tests/client-sdk/conftest.py

```diff
@@ -78,7 +78,7 @@ def provider_data():
 
 
 @pytest.fixture(scope="session")
-def llama_stack_client(provider_data):
+def llama_stack_client(provider_data, text_model_id):
```

Comment:
this isn't quite the right place to do registration because not all tests will be inference based. will figure out a better place.